modAionImpl/src/org/aion/zero/impl/sync/handler/BroadcastTxHandler.java

```diff
@@ -167,9 +167,14 @@ public final void receive(int _nodeIdHashcode, String _displayId, final byte[] _
                 }
             } catch (Exception e) {
                 // do nothing, invalid transaction from bad peer
+                System.out.println("castRawTx exception: " + e.toString());
```

Comment:
Suggest implementing this in a logger, and set the logging level to debug or trace. Otherwise I think people may complain

Reply:
2ad727b9227bf7cea433d2ee65b72c6930aa5dc1